starlette/middleware/__init__.py

```diff
@@ -14,7 +14,7 @@
 
 
 class _MiddlewareFactory(Protocol[P]):
-    def __call__(self, app: ASGIApp, *args: P.args, **kwargs: P.kwargs) -> ASGIApp: ...  # pragma: no cover
+    def __call__(self, app: ASGIApp, /, *args: P.args, **kwargs: P.kwargs) -> ASGIApp: ...  # pragma: no cover
```

Comment:
I find specifying positional only parameters in Protocols confusion, however I checked this case and I think it's correct:

```python
import typing

class Plain(typing.Protocol):
    def __call__(self, x: int, y: str) -> None: ...

class Positional(typing.Protocol):
    def __call__(self, a: int, /, y: str) -> None: ...


def foo(x: int, y: str) -> None:
    ...


def bar(x: int, /, y: str) -> None:
    ...


a: Plain = foo  # ok
b: Plain = bar # err
c: Positional = foo  # ok
d: Positional = bar # ok
```

Passes on https://mypy-play.net/?mypy=1.4.1&python=3.13&gist=8972af53af6b9d67e6c255a5de32db18